### ISSUE TITLE
Confetti backup ignores mutable values

### ISSUE THREAD

**Issue by ayalash** [getslash/confetti#17]
```
>>> from confetti import Config
>>> config = Config({'a': ['b']})
>>> config.root.a
['b']
>>> config.backup()
>>> config.root.a.append('c')
>>> config.root.a
['b', 'c']
>>> config.restore()
>>> config.root.a
['b', 'c']
```
The expectation is that after the `restore`:
```
>>> config.root.a
['b']
```

**Comment by ayalash** [comment]
@vmalloc I don't have permissions to this project, so I cannot assign it or set anything...

**Comment by vmalloc** [comment]
Added all Slash devs as collaborators

**Comment by ayalash** [comment]
@vmalloc Apparently @drrlvn already fixed this in commit to master 1.5 years ago:
```
commit bfd154b264ef0a76a490b7d66cc90fe26aadcced
Author: Dror Levin <spatz@psybear.com>
Date:   Sun Apr 24 17:28:10 2016 +0300

    Fix backup for mutable objects
    
    Backup now deep copies objects so that they are properly restored.
```
Can you please release a new version over master so we could depend on it?

**Comment by vmalloc** [comment]
Released 2.5.2 with the fix.
